thrift/src/main/thrift/entity.thrift

```diff
@@ -0,0 +1,26 @@
+namespace java com.gu.contententity.thrift.entity
+
+include "entities/person.thrift"
+include "entities/film.thrift"
+include "entities/game.thrift"
+include "entities/restaurant.thrift"
+
+enum EntityType {
+  PERSON = 0
+  FILM = 1
+  GAME = 2
+  RESTAURANT = 3
+}
+
+// A container for an entity
+struct Entity {
+  1: required string id         //Unique entity ID, e.g. person/theresa_may/12345
+  2: required EntityType entityType
+  3: optional string tagId      //Guardian Tag ID
```

Comment:
> I agree, but If not, how will you retrieve the entity when you only have a tag in Porter and you don't know it's type?

Using concierge endpoint on `entity` endpoint, we search what is the entity that match the tag.